src/routes/search/containers/SearchModalContainer.js

```diff
@@ -7,20 +7,36 @@ import type { StoreActionType } from '../../../modules/app/StoreActionType'
 import { withTheme } from 'styled-components/native'
 import createNavigateToCategory from '../../../modules/app/createNavigateToCategory'
 import SearchModal from '../components/SearchModal'
+import { CategoriesMapModel } from '@integreat-app/integreat-api-client'
+import type { NavigationScreenProp } from 'react-navigation'
 
-const mapStateToProps = (state: StateType, ownProps) => {
+type OwnPropsType = {| navigation: NavigationScreenProp<*> |}
+
+export type PropsType = {|
```

Comment:
Shouldn't this be the PropsTypes from `SearchModal`? At least it's a duplicate.

Reply:
Alright, I see. Many PropTypes now. Maybe rename to ConnectPropTypes. Not sure though.